components/search_results/messages_or_files_selector.tsx

```diff
@@ -26,7 +31,9 @@ export default function MessagesOrFilesSelector(props: Props): JSX.Element {
             <div className='buttons-container'>
                 <span
```

Comment:
nothing related to this PR, but semantically `span` should never have action listeners added to them.
Why are we not using `button`s here?

Reply:
Good point, I going to change it to button.

Reply:
Done